recipes-ni/sysconfig-settings/sysconfig-settings_1.0.bb

```diff
@@ -119,5 +122,5 @@ pkg_postinst_ontarget_${PN} () {
 	# Add the memory reservation for Base
 	/usr/local/natinst/bin/nirtcfg -s section=RtLinuxMemReserve,token=Base,value=24
 
-	${sysconfdir}/natinst/niselectsystemsettings postinst
+	/etc/natinst/niselectsystemsettings postinst
```

Comment:
Looks like this change went the wrong direction. Shouldn't it have left the `${sysconfdir}` variable alone?